onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -592,45 +598,10 @@ def validate(all_ref_outputs, all_outputs, rtol, atol, percent_mismatch):
                     logger.error(e)
                     return False, e
 
-    logger.info("ONNX Runtime outputs are similar to reference outputs!")
+    logger.debug("ONNX Runtime outputs are similar to reference outputs!")
     return True, None
 
 
-# not use for this script
-def cleanup_files():
-    files = []
-    p = subprocess.Popen(["find", ".", "-name", "test_data_set*", "-type", "d"], stdout=subprocess.PIPE)
-    stdout, sterr = p.communicate()
-    stdout = stdout.decode("ascii").strip()
-    files = files + stdout.split("\n")
-
-    p = subprocess.Popen(["find", ".", "-name", "*.onnx"], stdout=subprocess.PIPE)
-    stdout, sterr = p.communicate()
-    stdout = stdout.decode("ascii").strip()
-    files = files + stdout.split("\n")
-
-    p = subprocess.Popen(["find", ".", "-name", "*.gz"], stdout=subprocess.PIPE)
-    stdout, sterr = p.communicate()
-    stdout = stdout.decode("ascii").strip()
-    files = files + stdout.split("\n")
-
-    for f in files:
-        if "custom_test_data" in f:
-            logger.info(f)
-            continue
-        subprocess.Popen(["rm", "-rf", f], stdout=subprocess.PIPE)
-
-
-def remove_files(running_mode, path):
```

Comment:
This function never did anything (was never finished). This means that files generated by ort inference runs (e.g., TRT engines, profile data, subgraph dumps) were not cleaned up.

To fix, I updated the run_onnxruntime() function to dump everything in a temporary directory, which is automatically deleted after running.